.depcheckrc

```diff
@@ -8,6 +8,7 @@
     "@storybook/manager-webpack5",
     "@storybook/source-loader",
     "@types/*",
-    "jest-environment-jsdom"
+    "jest-environment-jsdom",
+    "geojson"
```

Comment:
depcheck was failing even though types are imported from `@types/geojson` (which is in `package.json`), not `geojson`